deployment/united-manufacturing-hub/templates/timescaledb/configmap.yaml

```diff
@@ -196,6 +196,35 @@ data:
     -- #2
     ALTER TABLE configurationTable ADD IF NOT EXISTS AutomaticallyIdentifyChangeovers BOOLEAN DEFAULT true;
 
+    -- #51 state table
```

Comment:
Instead of using a trigger here, can we make it use a constraing ?
So that we don't insert then remove, but check before inserting.

Reply:
Updated so that it checks new inputs before inserting. Also, a transaction that repairs statetable is added.